tools/ui-components/src/normalize.css

```diff
@@ -1,4 +1,12 @@
 /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+*,
+::before,
+::after {
+  /* Override the browser default border width in order to style individual border sides
+   * Ref: https://stackoverflow.com/a/76961084
+   */
+  border-width: 0;
+}
```

Comment:
Just a reminder that the minified bootstrap css includes normalize.css as well. If are changing things here we should technically be changing the same styles in bootstrap css. Otherwise, we would run into inconsistencies when replacing the component or when bootstrap is removed.